modules/claims_api/lib/claims_api/v2/disability_compensation_pdf_mapper.rb

```diff
@@ -20,10 +20,11 @@ class DisabilityCompensationPdfMapper # rubocop:disable Metrics/ClassLength
         4 => :convert_date_string_to_format_yyyy
       }.freeze
 
-      def initialize(auto_claim, pdf_data, target_veteran)
+      def initialize(auto_claim, pdf_data, target_veteran, file_number)
         @auto_claim = auto_claim
         @pdf_data = pdf_data
         @target_veteran = target_veteran
+        @file_number = file_number
```

Comment:
If the `target_veteran` is being passed in, do we need the `file_number`? Wouldn't `target_veteran.birls_id || target_veteran.mpi&.birls_id` work here?